src/api.js

```diff
@@ -28,19 +50,60 @@ function searchAlbums(query, page) {
       results: data.results
     }))
     .then(res => ({
-        ...res,
-        results: res.results.map(result => {
-          const splitTitle = result.title.split(' - ')
-          const [artist, title] = splitTitle
-          return {
-            masterId: result.master_id,
-            title,
-            artist,
-            thumb: result.thumb,
-            coverImage: result.cover_image
-          }
+      ...res,
+      results: res.results.map(result => {
+        const splitTitle = result.title.split(' - ');
+        const [artist, title] = splitTitle;
+        return {
+          masterId: result.master_id,
+          title,
+          artist,
+          thumb: result.thumb,
+          coverImage: result.cover_image
+        };
       })
-    }))
+    }));
 }
-  
-export { getTrendingAlbums, getAlbumsByMasterId, getAlbumsByGenre, searchAlbums }
+
+export function getAlbumDetails(albumID) {
+  return fetch(`https://api.discogs.com/masters/${albumID}?key=GimREdkHlKcSjALMSwEP&secret=RZbpExNDRyTdbTAaiVxiJpiYgOcydrMJ`)
+    .then(response => {
+      if (!response.ok) {
+        throw new Error('Error fetching album details.');
+      }
+      return response.json();
+    })
+    .then(data => {
+      console.log('Album details and data:', data);
+
+      const artistNames = Array.isArray(data.artist)
```

Comment:
I think this is breaking, an album is always showing unknown artist.

the array the api is hitting 'data.artists' not 'data.artist', just needs an extra s!

Reply:
great catch! was a fix